docs/source/tutorials.rst

```diff
@@ -15,4 +15,6 @@ The following are examples and notebooks on how to use SciKeras.
 
 * `Data Transformers <https://github.com/adriangb/scikeras/blob/master/notebooks/DataTransformers.ipynb>`_ - Use SciKeras data transformers to handle multi input, multi output and multidimensional Keras Model. `Run in Google Colab 💻 <https://colab.research.google.com/github/adriangb/scikeras/blob/master/notebooks/DataTransformers.ipynb>`_
 
+* `Autoencoder/Transformer Example <https://github.com/adriangb/scikeras/blob/master/notebooks/Autoencoders.ipynb>`_ - Learn how to make a Keras Autoencoder into a Scikit-Learn transformer. `Run in Google Colab 💻 <https://colab.research.google.com/github/adriangb/scikeras/blob/master/notebooks/Autoencoders.ipynb>`_
```

Comment:
Can these notebooks be rendered in HTML for the documentation? That way the user won't have to wait for GitHub to render them. (I think I've used [nb-sphinx][1] for this, but there are [other options][2]).

[2]:https://blog.jupyter.org/integrating-output-in-documentation-with-jupyter-sphinx-ecf569ddab85
[1]:https://nbsphinx.readthedocs.io/

Reply:
Do they take a long time for them to load? It takes ~1 sec for me. I'll consider it as part of a future documentation cleanup sprint (during which I also want to clean up all of the docstrings and document attributes).

Reply:
For me, it takes around 5 seconds (or more) for GitHub to render these notebooks (on two different browsers). That's a really long time in the web context. I'm sure the rendering time will grow with the notebook complexity (images/graphs, tables, etc).

I don't think it should be part of this PR either, just something I noticed.

Reply:
Noted. I opened #130 to track this.